includes/customer-functions.php

```diff
@@ -807,12 +813,6 @@ function edd_add_customer_email_address( $data ) {
  *                   false on error.
  */
 function edd_delete_customer_email_address( $customer_email_address_id = 0 ) {
```

Comment:
```suggestion
function edd_delete_customer_email_address( $customer_email_address_id ) {
```

Since a customer ID is required, let's remove the `0` default.